kimchi/src/circuits/gate.rs

```diff
@@ -361,8 +366,8 @@ impl<F: PrimeField + SquareRootField> CircuitGate<F> {
         };
 
         Ok(ArgumentWitness::<F> {
-            curr: witness_curr,
-            next: witness_next,
+            curr: witness_curr.to_vec(),
```

Comment:
This clone is unfortunate

Reply:
Oh yeah indeed, but if I revert the change, make them arrays again, and ArgumentWitness becomes generic over columns; then it will be avoided 😄